lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/mapping/testing/RuleBasedIdentifierMappingUtils.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.base.mapping.testing;
+
+import com.google.common.collect.ImmutableList;
+import io.airlift.units.Duration;
+import io.trino.plugin.base.mapping.IdentifierMappingRules;
+import io.trino.plugin.base.mapping.SchemaMappingRule;
+import io.trino.plugin.base.mapping.TableMappingRule;
+
+import java.nio.file.Path;
+import java.util.List;
+
+import static io.airlift.json.JsonCodec.jsonCodec;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.nio.file.Files.createTempFile;
+import static java.nio.file.Files.write;
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+
+public final class RuleBasedIdentifierMappingUtils
```

Comment:
This files appears as new in this commit

> Move test support classes to module source path
TestingEventListenerContext and TestingSystemAccessControlContext are used
for testing by dependent modules, so they should proper classes in the module.

the commit title says it is just _moves_, there should be no code in it.

Probably deletion of `lib/trino-plugin-toolkit/src/test/java/io/trino/plugin/base/mapping/RuleBasedIdentifierMappingUtils.java` was meant to be part of this commit.